src/Parser.js

```diff
@@ -94,8 +94,8 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 								parse(characters, root, reference, reference, props, rulesets, length, points, children)
 							else
 								switch (atrule === 99 && charat(characters, 3) === 110 ? 100 : atrule) {
-									// d m s
-									case 100: case 109: case 115:
+									// d l m s
```

Comment:
for completeness you should also add `export var LAYER = '@layer'` here:
https://github.com/thysultan/stylis/blob/ac5e81d7253c3d97e7121cba5bd36530c286ca55/src/Enum.js#L9-L20